README.adoc

```diff
@@ -1,103 +1,37 @@
 = Pi-gen Micro
 Richard Oliver <richard.oliver@raspberrypi.com>
 
-== Building
+pi-gen-micro is a system designed to build *tiny* embedded operating systems from the same package sources as Raspberry Pi OS.
+
+Why? Because this means you get the latest hardware support and updates in line with a widely-used and tested OS.
+
+== Installing
+
+=== Build the package
+Use the standard Debian `devhelper` package:
```

Comment:
Is this the correct package name? I don't have debuild and can't apt install 'devhelper' or 'debuild'.

apt install devscripts ?